src/bin/www.ts

```diff
@@ -77,14 +76,3 @@ function onError(error) {
       throw error;
   }
 }
-
-/**
- * Event listener for HTTP server "listening" event.
- */
-
-function onListening() {
-  const addr = server.address();
```

Comment:
these vars weren't being used, so we are left with an empty function. I've tested the app and it works without this.